wrappers/swift/SDWLogManager.swift

```diff
@@ -0,0 +1,182 @@
+    }
+
+    /**
+    Retrieves a new instance of `Logger` for logging telemetry events. Initializes the telemetry logging system,
+    if not already initialized, with the default configuration using the specified tenant token and source.
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - withSource: A `String` that contains the name of the source of events.
+
+    - Returns:
+        - A `Logger` instance that points to the logger for the specified tenant token and source.
+    */
+    public static func loggerWithTenant(tenantToken: String!, withSource source: String! = "") -> Logger? {
+        if let odwLogger = ODWLogManager.logger(withTenant: tenantToken, source: source) {
+            return Logger(logger: odwLogger)
+        } else {
+            return nil
+        }
+    }
+
+    /**
+    Retrieves a new instance of `Logger` for logging telemetry events.
+    This method expects that the logging system has already been initialized.
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - withSource: A `String` that contains the name of the source of events.
+        - config: A `Dictionary` that contains a custom configuration.
+
+    - Returns:
+        - A `Logger` instance that points to the logger for the specified tenant token and source.
+    */
+    public static func loggerWithTenant(tenantToken: String!, withSource source: String!, withConfig config: ODWLogConfiguration!) -> Logger?{
+        if let odwLogger = ODWLogManager.logger(withTenant: tenantToken, source: source, withConfig: config) {
+            return Logger(logger: odwLogger)
+        } else {
+            return nil
+        }
+    }
+
+    /**
+    Retrieves a new instance of `Logger` for logging telemetry events.
+
+    - Note:
+        It requires to previously call `loggerWithTenant` method.
+
+    - Parameters:
+        - source: A `String` that contains the name of the source of events sent by this logger instance.
+
+    - Return:
+        - A `Logger` instance that points to the logger for source.
+    */
+    public static func loggerForSource(source: String!) -> Logger? {
+        if let odwLogger = ODWLogManager.logger(forSource: source) {
+            return Logger(logger: odwLogger)
+        } else {
+            return nil
+        }
+    }
+
+    /// Attempts to send any pending telemetry events that are currently cached either in memory, or on disk.
+    /// Use this method if your event can't wait for automatic timed upload.
+    public static func uploadNow() {
+        ODWLogManager.uploadNow()
+    }
+
+    /// Flushes pending telemetry events from memory to disk (to reduce possible data loss) and returns the flush operation's status.
+    public static func flush() throws -> ODWStatus {
+        do {
+            return ODWLogManager.flush()
+        } catch let exception as NSException {
+            if (LogConfiguration.surfaceCppExceptions()) {
```

Comment:
Isn't this bubbling-up/eating behavior already enforced by Obj-C layer? If that's the case, then I don't see why we need to add it here also

Reply:
Yes. Similar logic is being used at objc layer while wrapping cpp exceptions. I think we can remove this one and let the objc exceptions be handled by app